### ISSUE TITLE
Vaults not showing content

### ISSUE THREAD

**Issue by Lumiclipse** [atomizer/muledump#135]
Whenever I open muledump the content in my vaults don't show and only shows empty boxes, any solutions? I've already tried to use this solution, http://www.mpgh.net/forum/showthread.php?t=1260566, but it just breaks my muledump and doesn't load any of my accounts.

**Comment by KnightDoom** [comment]
Patch provided by todd is functional and fixes Muledump output. Please ensure to reload your browser if you have to. 

I have zipped my file below. 
[mule.zip](https://github.com/atomizer/muledump/files/1015848/mule.zip)

**Comment by Lumiclipse** [comment]
Thank you so much! You're my savior ^^

**Comment by Nightfirecat** [comment]
Huh... I'll get a PR opened for this over the weekend if the fix is really that easy. I wonder what changed to require that.

**Comment by AlexMooney** [comment]
Hidden at the bottom of these patch notes: https://www.reddit.com/r/RotMG/comments/6bx8am/patch_notes_277x14_20170518/

> And don’t worry, Muledump will still be usable with the changes, simply add &muleDump=true to the URL parameters.

**Comment by Invertisment** [comment]
Wouldn't this mean that DECA will find out every mule account?

**Comment by tischepe** [comment]
Not really, just accounts that muledump is used on, muledump is useful even for people who don't use mules

**Comment by Invertisment** [comment]
I believe that about 95% of those accounts will be the mules. And if you have used muledump and you are lv1, then you are a mule. Win.

**Comment by Nightfirecat** [comment]
I'm fairly confident that they could have discovered that information easily enough without this flag, given that requests before would have been akin to a player logging in to the main menu screen, then immediately logging out. Kinda strange for a normal player to do so on the regular while never actually playing the game. 🤔
